addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/internal/BigAssFanHandlerFactory.java

```diff
@@ -23,6 +26,13 @@
  * @author Mark Hilbush - Initial contribution
  */
 public class BigAssFanHandlerFactory extends BaseThingHandlerFactory {
+    private Logger logger = LoggerFactory.getLogger(BigAssFanHandlerFactory.class);
+
+    @Override
+    protected void activate(ComponentContext componentContext) {
```

Comment:
I used the following to get the version in a handler:
        `Version version = FrameworkUtil.getBundle(this.getClass()).getVersion();`